tests/examples/test_contrib/prometheus/test_prometheus_exporter_example.py

```diff
@@ -20,7 +20,8 @@ def clear_collectors() -> None:
 @pytest.mark.parametrize(
     "group_path, route_path, route_template, expected_path",
     [
-        (True, "/test/litestar", "test/{name:str}", "/test/{name}"),
```

Comment:
The removed test case should still pass, otherwise we'd have to consider this a breaking change

Reply:
It does pass.
It was a duplicate test case, see https://github.com/litestar-org/litestar/blob/36d3faa8109d90de2a714e9cc0b21cf0334b04d9/tests/examples/test_contrib/prometheus/test_prometheus_exporter_example.py#L25